fineract-core/src/main/java/org/apache/fineract/infrastructure/security/utils/SQLInjectionValidator.java

```diff
@@ -180,4 +143,17 @@ private static void patternMatchSqlInjection(String sqlSearch, String lowerCaseS
         }
     }
 
+    private static void validateSQLCommand(String lowerCaseSQL, String[] commands, SQLCommandCondition SQLCondition) {
+        for (String command : commands) {
+            if (SQLCondition.checkCondition(command, lowerCaseSQL)) {
+                throw new SQLInjectionException();
+            }
+        }
+    }
+
+    private static void validateSQLCommands(String lowerCaseSQL, List<String[]> commandsList, SQLCommandCondition conditions) {
```

Comment:
Please rename `conditions` hence it is just one condition. (ex: condition)

Reply:
@adamsaghy Yes, you're right. I changed it. Please check 🙇